setup.py

```diff
@@ -23,7 +23,7 @@
 
 install_requires = [
     'six>=1.7.2',
-    'isbnlib>=3.5.6',
+    'isbnid==0.3.4',
```

Comment:
It's something weird with the release number on pypi and the one in egg info. I couldn't make the build pass unless I pinned the version to this one.